packages/ssh-tunnel/src/index.ts

```diff
@@ -70,57 +73,16 @@ export class SshTunnel extends EventEmitter {
 
     this.sshClient = new SshClient();
 
+    this.sshClient.on('close', () => {
```

Comment:
https://github.com/mscdex/ssh2/issues/491#issuecomment-258041615

> Yes, the 'close' event will always be called when a socket is closed, whether a connection was ended gracefully or closed due to error. Speaking of which, you'll want to add an 'error' event handler for the connection in case of errors, to prevent the process from dying by default.